### DISCUSSION TITLE
Ckeditor sourceEditing button

### DISCUSSION THREAD

**Discussion by romain35132** [sulu/sulu#6468] [Q&A]
Hi, 

I'm currently upgrading a website for my employeer, and i saw that the button "Source" as been replaced by the "code" one (which as the same image). I followed the cookbook https://ckeditor.com/docs/ckeditor5/latest/features/source-editing.html to install and enable it on my environnement but I have some errors when I try to run it. Does someone already have succeeded it ?

Thx for the answer :)

**Comment by luca-rath** [comment]
No, I have never implemented this plugin. Unfortunately we can't help you if you don't tell us what errors you get when you try to run it

**Reply by romain35132** [reply]
Hi, ckeditor5-source-editing was the good plugin to install. (https://ckeditor.com/docs/ckeditor5/latest/features/source-editing.html)
